### PR TITLE
Improve the err msg about the subscription-manager missing

### PR DESCRIPTION
**Pull request by bocekm**
Ccing @Turivniy.

### CODE REVIEW

File: convert2rhel/subscription.py [hunk 1]
```diff
@@ -127,12 +127,12 @@ def install_subscription_manager():
...
     sm_dir = os.path.join(utils.data_dir, "subscription-manager")
     if not os.path.isdir(sm_dir) or not os.listdir(sm_dir):
-        loggerinst.critical("The {0} directory does not exist or is empty."
-                            " Consider using the --disable-submgr option."
-                            " Before using it it's necessary to define a RHEL"
-                            " repo in /etc/yum.repos.d/ and pass its repoid to"
-                            " the convert2rhel through the --enable-repo"
-                            " option.".format(sm_dir))
+        loggerinst.critical("The %s directory does not exist or is empty."
+                            " Using the subsription-manager is not documented"
```
Comment (Turivniy) [inline convert2rhel/subscription.py]: Typo: in subscription-manager

Comment (Turivniy) [inline convert2rhel/subscription.py]: Typo in subscription-manager
please, update

File: convert2rhel/subscription.py [hunk 2]
```diff
@@ -127,12 +127,12 @@ def install_subscription_manager():
...
+                            " yet. Please use the --disable-submgr option."
```
Comment (Turivniy) [inline convert2rhel/subscription.py]: Typo in  --disable-submit
please, update

Comment (bocekm) [reply convert2rhel/subscription.py]: This option is actually correct - no typo: https://github.com/oamg/convert2rhel/blob/master/convert2rhel/toolopts.py#L135

File: convert2rhel/subscription.py (hunk 1 above)
Comment (bocekm) [reply convert2rhel/subscription.py]: Thanks. Fixed.

File: convert2rhel/subscription.py (hunk 2 above)
Comment (Turivniy) [reply convert2rhel/subscription.py]: oh, sorry =)

**Pull request merged**